f/dynamicViewDistance/f_setViewDistanceInit.sqf

```diff
@@ -13,11 +13,16 @@ if (!isDedicated && (isNull player)) then
 
 // Add event handlers:
 // Note: Using PlayerViewChanged instead of GetInMan and GetOutMan to cover UAVs as well.
-f_ehIndex_dynamicViewDistance_0 = player addEventHandler ['SeatSwitchedMan',   f_fnc_ehSetViewDistance];
-f_ehIndex_dynamicViewDistance_1 = addMissionEventHandler ['PlayerViewChanged', f_fnc_ehSetViewDistance];
+// Note: GetInMan is still needed due to a bug where PlayerViewChanged doesn't fire for
+//       non-server players when they enter a vehicle they haven't entered before.
+//       Because of this, when entering a vehicle the code is sometimes run twice.
+f_ehIndex_dynamicViewDistance_0 = addMissionEventHandler ['PlayerViewChanged', f_fnc_ehSetViewDistance];
+f_ehIndex_dynamicViewDistance_1 = player addEventHandler ['SeatSwitchedMan',   f_fnc_ehSetViewDistance];
+f_ehIndex_dynamicViewDistance_2 = player addEventHandler ['GetInMan',          f_fnc_ehSetViewDistance];
```

Comment:
This causes `f_fnc_ehSetViewDistance` to get called twice when entering a vehicle (after the first), but unless there is a quick fix for it, I'm fine with it. The second call doesn't have any effect and hopefully this  bug gets fixed and makes this unnecessary.